app/src/main/java/com/cornellappdev/android/eatery/MainListFragment.java

```diff
@@ -88,10 +88,16 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
     mSwipesButton.setOnClickListener(this);
     mBrbButton.setOnClickListener(this);
 
+    // Resetting the buttons on page reload
```

Comment:
I feel like we could make this into a method and name is resetButtons or sthng. More intuitive than explaining with a comment